Update automatically-generated-release-notes.md (#18156)
github/docs: content/repositories/releasing-projects-on-github/automatically-generated-release-notes.md

```
{% endraw %}

> A configuration for a repo that doesn't tag Pull Requests but wants to separate out Dependabot automated PRs in release notes (note: `labels: '*'` is required to display a catchall category)

Reviewer: we avoid using `repo` and `PR` in the docs, so suggesting this:
```suggestion
A configuration for a repository that doesn't tag pull requests but where we want to separate out Dependabot automated pull requests in release notes (`labels: '*'` is required to display a catchall category)
```

> A configuration for a repository that doesn't tag pull requests but where we want to separate out Dependabot automated pull requests in release notes (`labels: '*'` is required to display a catchall category)

Reviewer: Use product variable
```suggestion
A configuration for a repository that doesn't tag pull requests but where we want to separate out {% data variables.product.prodname_dependabot %} automated pull requests in release notes (`labels: '*'` is required to display a catchall category)
```
